packages/admin/src/api.ts

```diff
@@ -160,22 +163,18 @@ export class AdminClient extends BaseApiClient {
     });
   }
 
-  public async archiveProposal(contractId: string, proposalId: string): Promise<ProposalResponseWithUrl> {
+  public async archiveProposal(_: string, proposalId: string): Promise<ProposalResponseWithUrl> {
     return this.apiCall(async (api) => {
-      const response = (await api.put(`/contracts/${contractId}/proposals/${proposalId}/archived`, {
-        archived: true,
-      })) as ProposalResponse;
+      const response = (await api.put(`proposals/archive/${proposalId}`)) as ProposalResponse;
       return { ...response, url: getProposalUrl(response) };
-    });
+    }, 'v2');
   }
 
-  public async unarchiveProposal(contractId: string, proposalId: string): Promise<ProposalResponseWithUrl> {
+  public async unarchiveProposal(_: string, proposalId: string): Promise<ProposalResponseWithUrl> {
```

Comment:
`contractId` is no longer required but we keep the param for backwards compatibility